src/sql/src/plan/transform_ast.rs

```diff
@@ -272,43 +288,55 @@ impl<'a> FuncRewriter<'a> {
                 distinct,
                 over: None,
             }) => {
-                let name = normalize::unresolved_item_name(name.clone()).ok()?;
-                if let Some(database) = &name.database {
-                    // If a database name is provided, we need only verify that
-                    // the database exists, as presently functions can only
-                    // exist in ambient schemas.
-                    if let Err(e) = self.scx.catalog.resolve_database(database) {
-                        self.status = Err(e.into());
+                let name = match name {
+                    ResolvedItemName::Item {
+                        qualifiers,
+                        full_name,
+                        ..
+                    } => {
+                        if &qualifiers.schema_spec
+                            != self
+                                .scx
+                                .catalog
+                                .resolve_schema(None, "pg_catalog")
```

Comment:
Should use `PG_CATALOG_SCHEMA` in all places instead of hard coding a string.

Reply:
Did this and a little bit more by moving these consts into `mz_repr::namespaces`